Podcast/PlayerViewController.swift

```diff
@@ -127,6 +123,8 @@ class PlayerViewController: TabBarAccessoryViewController, PlayerDelegate, Playe
         if animated {
             UIView.animate(withDuration: 0.5, animations: {
```

Comment:
We do this animation a lot and its all with 0.5 seconds so let's put 0.5 seconds as a constant above and replace all the 0.5's in the animation code ... another alternative, create a function in here `animatePlayer(animations: () -> (), completion (x) -> ())` that automatically encodes in the 0.5 animation duration